app/assets/javascripts/components/sampleUpload.jsx

```diff
@@ -41,8 +42,9 @@ class SampleUpload extends React.Component {
       successMessage: '',
       project: 'Select a Project',
       projectId: null,
-      job_queue: null,
-      memory: null,
+      job_queue: 'aegea_batch',
```

Comment:
You don't have to set the default values for job_queue and memory any more.  It would pick up from host_genomes if it's not set. so keep line 44 and 45